src/views/mentorship/mentorship.tsx

```diff
@@ -1,9 +1,9 @@
-import { MentorshipHero } from './mentorship-hero/ui/mentorship-hero';
-import { DetailsMentorship } from './ui/details/details';
-import { MentorActivities } from './ui/mentor-activities';
-import { MentorshipCourses } from './ui/mentorship-courses/mentorship-courses';
-import { MentorsAfterRegister } from '@/views/mentorship/ui/mentors-after-register';
+import { MentorActivities } from '../../widgets/mentorship-activities';
```

Comment:
can we use alias import here?

```suggestion
import { MentorActivities } from '@/widgets/mentorship-activities';
```